node/src/command.rs

```diff
@@ -144,6 +144,10 @@ pub fn run() -> sc_cli::Result<()> {
                     .into())
             }
         }
+        Some(Subcommand::PeerIdHex(peer_id)) => {
+            // println!("Converting Peer ID in base58 to hex");
```

Comment:
Please remove the commented out code here.

Reply:
Would remove it.